eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -1,46 +1,67 @@
 param(
   [Parameter(Mandatory = $true)]
-  $RepoOwner,
+  # Please use the RepoOwner/RepoName format: e.g. Azure/azure-sdk-for-java
+  $Repo,
   # Use this if a pull request might have been opened from one repo against another.
   # E.g Pull request opened from azure-sdk/azure-sdk prBranch --> Azure/azure-sdk baseBranch
-  $ForkRepoOwner,
-  [Parameter(Mandatory = $true)]
-  $RepoName,
+  $ForkRepoName,
   [Parameter(Mandatory = $true)]
   $BranchPrefix,
-  [Parameter(Mandatory = $true)]
-  $AuthToken
+  # Date format: Tuesday, April 12, 2022 1:36:02 PM
+  $LastCommitOlderThan
 )
 
 . (Join-Path $PSScriptRoot common.ps1)
 
-LogDebug "Operating on Repo [ $RepoName ]"
-try{
-  $branches = (Get-GitHubSourceReferences -RepoOwner $RepoOwner -RepoName $RepoName -Ref "heads/$BranchPrefix" -AuthToken $AuthToken).ref 
```

Comment:
What other properties were returned by Get-GitHubSourceReferences? I see we only take the ref but does the ref url or other method exist to get the commit timestamp?

Reply:
The API returns a commit sha url. We can call the url directly to get the commit timestamp.

For example,
Get-GitHubSourceReferences returns:
```
[
  {
    "ref": "refs/heads/main",
    "node_id": "MDM6UmVmNjM2MTQwNzk6cmVmcy9oZWFkcy9tYWlu",
    "url": "https://api.github.com/repos/Azure/azure-docs-sdk-java/git/refs/heads/main",
    "object": {
      "sha": "31a229aa7319e18123ff1b2bc6bc13f0685bd73d",
      "type": "commit",
      "url": "https://api.github.com/repos/Azure/azure-docs-sdk-java/git/commits/31a229aa7319e18123ff1b2bc6bc13f0685bd73d"
    }
  }
```

Invoke the api `https://api.github.com/repos/Azure/azure-docs-sdk-java/git/commits/31a229aa7319e18123ff1b2bc6bc13f0685bd73d` will return:
```
{
  "sha": "31a229aa7319e18123ff1b2bc6bc13f0685bd73d",
  "node_id": "C_kwDOA8qsf9oAKDMxYTIyOWFhNzMxOWUxODEyM2ZmMWIyYmM2YmMxM2YwNjg1YmQ3M2Q",
  "url": "https://api.github.com/repos/Azure/azure-docs-sdk-java/git/commits/31a229aa7319e18123ff1b2bc6bc13f0685bd73d",
  "html_url": "https://github.com/Azure/azure-docs-sdk-java/commit/31a229aa7319e18123ff1b2bc6bc13f0685bd73d",
  "author": {
    "name": "azure-sdk",
    "email": "azuresdk@microsoft.com",
    "date": "2022-04-25T23:34:55Z" # date is right here
  },
  "committer": {
    "name": "azure-sdk",
    "email": "azuresdk@microsoft.com",
    "date": "2022-04-25T23:34:55Z"
  },
  ...
}
```

Reply:
That's why we have to pass the entire response instead of ref only.

Reply:
OK I just wanted to verify it was giving you the same thing which it seems to be doing so we can still use the rest apis.